autopush/endpoint.py

```diff
@@ -254,6 +278,43 @@ def _db_error_handling(self, d):
         d.addErrback(self._response_err)
         return d
 
+    def _store_auth(self, jwt, result):
+        if jwt.get('exp') and jwt['exp'] < time.time():
+            raise VapidAuthException("Invalid bearer token: Auth expired")
+        self._client_info.update(jwt)
```

Comment:
How many entries are in the jwt dict? Do all of them need to be in the client_info? Might be useful to nest them in a sub-dict in case its possible any of the jwt entries might smash an existing entry.